### PR TITLE
ci: update travis config

### PR DESCRIPTION
**Pull request by arekkas**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +13,9 @@ matrix:
 
 install:
    - go get github.com/mattn/goveralls golang.org/x/tools/cmd/cover github.com/pierrre/gotestcover github.com/Masterminds/glide
-   - glide up
+   - go get -t ./...
```
Comment (otraore) [inline .travis.yml]: This is why the builds are failing, why don't you remove this? All the dependencies should be installed on glide install

Comment (arekkas) [inline .travis.yml]: working on it, have to resolve some other packages first ;)

Comment (arekkas) [inline .travis.yml]: this is why it's not working: https://github.com/Masterminds/glide/issues/542

**Comment by arekkas** [comment]
what a journey...

**Pull request merged**